composer.json

```
@@ -1,68 +1,31 @@
 {
-    "type": "project",
-    "license": "proprietary",
-    "minimum-stability": "stable",
-    "prefer-stable": true,
-    "require": {
-        "php": ">=8.2",
-        "ext-ctype": "*",
-        "ext-iconv": "*",
-        "symfony/console": "7.1.*",
-        "symfony/dotenv": "7.1.*",
-        "symfony/flex": "^2",
-        "symfony/framework-bundle": "7.1.*",
-        "symfony/runtime": "7.1.*",
-        "symfony/yaml": "7.1.*"
-    },
-    "require-dev": {
-    },
-    "config": {
-        "allow-plugins": {
-            "php-http/discovery": true,
-            "symfony/flex": true,
-            "symfony/runtime": true
-        },
-        "sort-packages": true
-    },
+    "name": "sylius/gmv-bundle",
+    "description": "Lightweight local command to calculate the Sylius instance GMV within a specific period",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "authors": [
+    ],
     "autoload": {
         "psr-4": {
-            "App\\": "src/"
+            "Sylius\\GmvBundle\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "App\\Tests\\": "tests/"
+            "Tests\\Sylius\\GmvBundle\\": "tests/"
         }
     },
-    "replace": {
-        "symfony/polyfill-ctype": "*",
-        "symfony/polyfill-iconv": "*",
-        "symfony/polyfill-php72": "*",
-        "symfony/polyfill-php73": "*",
-        "symfony/polyfill-php74": "*",
-        "symfony/polyfill-php80": "*",
-        "symfony/polyfill-php81": "*",
-        "symfony/polyfill-php82": "*"
-    },
-    "scripts": {
-        "auto-scripts": {
-            "cache:clear": "symfony-cmd",
-            "assets:install %PUBLIC_DIR%": "symfony-cmd"
-        },
-        "post-install-cmd": [
-            "@auto-scripts"
-        ],
-        "post-update-cmd": [
-            "@auto-scripts"
-        ]
-    },
-    "conflict": {
-        "symfony/symfony": "*"
+    "require": {
+        "php": "^8.1",
+        "symfony/dependency-injection": "^5.4 || ^6.4",
+        "symfony/http-kernel": "^5.4 || ^6.4",
+        "symfony/framework-bundle": "^5.4 || ^6.4",
+        "symfony/console": "^5.4 || ^6.4",
+        "sylius/core": "~1.12.0 || ~1.13.0",
+        "sylius/money-bundle": "~1.12.0 || ~1.13.0"
```

I wonder if the Core component is enough here 🤔 
```suggestion
        "sylius/core-bundle": "~1.12.0 || ~1.13.0",
```